### PR TITLE
Update log messges for `extract_optional_permissions`

### PR DESCRIPTION
**Pull request by bobsilverberg**
Just a quick follow-up to #15076 to update the log messages

### CODE REVIEW

File: src/olympia/files/management/commands/extract_optional_permissions.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ def handle(self, *args, **options):
         ).order_by('pk')
         pks = files.values_list('pk', flat=True)
 
-        log.info('Using %s file pks to extract permissions' % pks.count())
+        log.info('Using %s file pks to extract permissions, max pk: %s' %
+                 (pks.count(), pks[len(pks) - 1]))
```
Comment (eviljeff) [inline src/olympia/files/management/commands/extract_optional_permissions.py:22]: `pks[-1]` is the same as `pks[len(pks) - 1]`

File: src/olympia/files/tasks.py [hunk 2]
```diff
@@ -22,8 +22,13 @@
 @task
 @use_primary_db
 def extract_optional_permissions(ids, **kw):
-    log.info('[%s@%s] Extracting permissions from Files, starting at id: %s...'
-             % (len(ids), extract_optional_permissions.rate_limit, ids[0]))
+    log.info(
+        '[%s@%s] Extracting permissions from Files, from id: %s to id: %s...'
+        % (
+            len(ids), extract_optional_permissions.rate_limit, ids[0],
+            ids[len(ids) - 1]
```
Comment (eviljeff) [inline src/olympia/files/tasks.py:29]: same here.

File: src/olympia/files/management/commands/extract_optional_permissions.py (hunk 1 above)
Comment (bobsilverberg) [reply src/olympia/files/management/commands/extract_optional_permissions.py:22]: I tried that. I got "Negative indexing is not supported" :-(

Comment (eviljeff) [reply src/olympia/files/management/commands/extract_optional_permissions.py:22]: ah, must be a queryset thing.  I feel `len(pks)` should be `pks.count()` again though, or both `len` - seems inconsistent to have the same thing two different ways in one statement.

**Pull request merged**